docs/optimizations/CheckpointFiles.md

```diff
@@ -4,27 +4,28 @@
 
 Available in Cromwell 55 and higher.
 
-This optimization hopes to resolve the issue of your worker VM being preempted 9 hours and 55 minutes into the runtime of
+This optimization aims to resolve the issue of your worker VM being preempted 9 hours and 55 minutes into the runtime of
```

Comment:
Even more confidence!
```suggestion
This optimization addresses the issue of your worker VM being preempted 9 hours and 55 minutes into the runtime of
```

Reply:
That extra confidence might be overpromising though. This addresses the issue in particular cases, and only if you can structure your task in the requisite way.

Reply:
How about if we said "provides a way to mitigate the problem of long-running tasks getting preempted partway through execution" ? Or something to that effect?